srcpkgs/git-cola/template

```diff
@@ -1,14 +1,13 @@
 # Template file for 'git-cola'
 pkgname=git-cola
-version=3.7
-revision=2
+version=3.8
+revision=1
 build_style=python3-module
-pycompile_dirs="usr/share/git-cola/lib"
```

Comment:
why? there is still lot of code there

Reply:
My mistake 👍

Reply:
Fixed, thank you! I build 3.8 but forgot to install... So my "testing" was with the previous version :(